static/app/components/gridEditable/styles.tsx

```diff
@@ -53,7 +53,7 @@ export const Body = styled(({children, ...props}: any) => (
   </Panel>
 ))`
   overflow-x: auto;
-  overflow-y: hidden;
+  overflow-y: ${props => (props?.showVerticalScrollbar ? 'auto' : 'hidden')};
```

Comment:
Shouldn't you destructure off `showVerticalScrollbar` here or did you mean to pass it to `Panel` via `...props` too?